pkgdb/tests/environment.cc

```diff
@@ -850,17 +850,16 @@ test_createLockfile_error()
       Lockfile actualLockfile = environment.createLockfile();
       return false;
     }
-  catch ( flox::FloxException & err )
+  catch ( const flox::resolver::ResolutionFailureException & )
     {
-      EXPECT_EQ(
-        err.what(),
-        std::string(
```

Comment:
suggestion nonblocking: if you make the changes I suggested about `msg` and `groupMsg`, I would rather leave this in

Reply:
Maybe we could check for the error category ( enum ) instead of the actual message?

Really what I was trying to avoid was writing test cases that are sensitive to small changes to those messages.